pkg/admin/facade/interaction.go

```diff
@@ -43,6 +43,13 @@ func (adminAPIOp) BypassPublicSignupDisabled() bool {
 	return true
 }
 
+// BypassLoginIDEmailByPassBlocklistAllowlist indicates Admin API operations
+// bypass email domains blocklist allowlist checking; Admin API can create
+// email login id that doesn't have blocklist allowlist restriction
+func (adminAPIOp) BypassLoginIDEmailByPassBlocklistAllowlist() bool {
```

Comment:
It seems that the `ByPass` in the middle is extra. Would `BypassLoginIDEmailBlocklistAllowlist` a better name?